src/python/module/smartredis/dataset.py

```diff
@@ -169,3 +176,164 @@ def get_meta_strings(self, name):
         """
         typecheck(name, "name", str)
         return self._data.get_meta_strings(name)
+
+
+    def import_from_xarray(self,xr):
+        # do the conversion we discussed here
+ 
+        # data variables
+        ## convert xr to a dict -- easier to separate vars -- already in a form that is right for reconstruction 
+        ##  could do differently --> direclty accessing from xarray 
+        
+        # XARRAY TO DICTIONARY 
+        dict = xr.to_dict()   
+    
+        #to_da = xr.data 
+        #att= xr.attrs  
+        # when adding metastring and tensor should keep track of how things are named for reconstruction  
+       # print(dict) 
+      #  print(dict.keys())   #(['dims', 'attrs', 'data', 'coords', 'name'])
+      #  print(dict["dims"])    #('x', 'y', 'time') 
+      #  print(dict["attrs"])  # {'description': 'Ambient temperature.', 'units': 'degC'}
+      
```

Comment:
rather than commenting out all these lines, you could indent them and put in a line like `if debug:`